src/odemis/driver/andorcam2.py

```diff
@@ -422,6 +431,7 @@ def __getitem__(self, name):
 20099: "DRV_BINNING_ERROR",
 20100: "DRV_INVALID_AMPLIFIER",
 20101: "DRV_INVALID_COUNTCONVERT_MODE",
+20102: "DRV_USB_INTERRUPT_ENDPOINT_TIMEOUT",
```

Comment:
With such a long list it maybe makes sense to create a file which contains just constants like this one? (ini or just python)
I don't think it required, and maybe it is not that Pythonic, @pieleric what is your view on this?

Reply:
I agree, sounds good.